Salesforce Adobe Target Actions Public Beta Changes (#2780)
segmentio/segment-docs: src/connections/destinations/catalog/actions-adobe-target-cloud/index.md

- [Adobe Target Web](/docs/connections/destinations/catalog/actions-adobe-target-web/)
- [Adobe Target Cloud Mode](/docs/connections/destinations/catalog/actions-adobe-target-cloud/)

> info ""
> The Adobe Target Cloud Mode destination is in beta and is in active development. Some functionality may change before it becomes generally available.

> > success ""

Reviewer: ```suggestion
> success "Good to know"
```

> > This document is about a feature which is in beta. This means that the Adobe Target Cloud Mode destination is in active development, and some functionality may change before it becomes generally available.

Reviewer: ```suggestion
> The Adobe Target Cloud Mode destination is in beta and is in active development. Some functionality may change before it becomes generally available.
```

> > **Good to know**: This page is about Segment's Adobe Target Cloud Mode destination. There's also a page about Segment's [Adobe Target Web destination](/docs/connections/destinations/catalog/actions-adobe-target-web/). **In order to use Adobe Target Cloud Mode, you must have a parallel web integration with Adobe Target as profiles can only be created by the Adobe Target `at.js` web script.**

Reviewer: ```suggestion
> This page is about Segment's Adobe Target Cloud Mode destination. There's also a page about Segment's [Adobe Target Web destination](/docs/connections/destinations/catalog/actions-adobe-target-web/). **In order to use Adobe Target Cloud Mode, you must have a parallel web integration with Adobe Target as profiles can only be created by the Adobe Target `at.js` web script.**
```

## Viewing Segment data in Adobe Target
To view and use your Segment data in Adobe Target, navigate to **Adobe Target > Audiences > Create Audience > Add Rule**.

- Profile Attributes appear under **Visitor Profile** attributes.
- Page Parameters appear under **Custom** attributes. Fields have `page.` prepended to the key.

Adobe Target Audiences can be used in Activities, such as A/B Testing and Experience Targeting. Please note that while Standard and Premium Adobe Target packages allow access to the SDK and API, certain personalization functionality may only be available with Adobe Target Premium. 

## FAQ
### Why am I getting a `Profile Not Found` error?

> The Adobe Target API can only be used for profile updates. Profiles must be created on the web first, either using Segment’s [Adobe Target Web destination](/docs/connections/destinations/catalog/actions-adobe-target-web/) or a native implementation of `at.js`. Please ensure you are creating profiles on the web first.

Reviewer: ```suggestion
The Adobe Target API can only be used for profile updates. You must first create profiles on the web by using either Segment’s [Adobe Target Web destination](/docs/connections/destinations/catalog/actions-adobe-target-web/) or a native implementation of `at.js`. Please ensure you create profiles on the web first.
```
